tools/install_envs.sh

```diff
@@ -0,0 +1,12 @@
+# script must be executed from root git directory
+pip install -r requirements.txt
+pip install -r test_requirements.txt
+pip install -r src/server/server/requirements.txt
```

Comment:
appengine_env is the environment used to start the flask server as you can see in tools/start_server.sh, to ensure the server is started as in appengine.

But unit tests task and pylint task are run once for all tests, including unit tests of server, and thus share the same env and must have access to server requirements.